src/firecracker/src/api_server/request/balloon.rs

```diff
@@ -51,8 +51,8 @@ pub(crate) fn parse_patch_balloon(
 
 #[cfg(test)]
 mod tests {
+    use super::super::super::parsed_request::tests::vmm_action_from_request;
```

Comment:
nit: For these, I think `use crate::api_server::parsed_request::...` is not only more readable, but also shorter xP
```suggestion
    use crate::api_server::parsed_request::tests::vmm_action_from_request;
```

Reply:
Agreed, I missed this one when I simplified the rest. Will change it